.github/workflows/integration_test.yml

```diff
@@ -219,7 +219,9 @@ jobs:
             sleep 1
           done
       - name: Run Java Integration Tests
-        run: ./mvnw --errors --batch-mode --show-version -P${{ matrix.integration-test-profile }} ${{ inputs.maven_args_tests }} ${{ inputs.test-goal }}
+        if: inputs.test-goal == 'test verify' 
```

Comment:
How about `if: inputs.test-goal != '' ` instead? That way, we can specify custom goals when we want to.

Reply:
Fixed